tests/CronofyTest.php

```diff
@@ -307,6 +307,39 @@ public function testDeleteEvent()
         $this->assertNotNull($actual);
     }
 
+    public function testBulkDeleteEvents()
+    {
+        $params = ["calendar_ids" => ["cal_123"], "delete_all" => true];
+
+        $http = $this->createMock(HttpRequest::class);
+        $http->expects($this->once())
+            ->method('httpDelete')
+            ->with(
+                $this->equalTo('https://api.cronofy.com/v1/events'),
+                $this->equalTo($params),
+                $this->equalTo([
+                    'Authorization: Bearer accessToken',
+                    'Host: api.cronofy.com',
+                    'Content-Type: application/json; charset=utf-8',
+                ])
+            )
+            ->will($this->returnValue(["{'foo': 'bar'}", 200]));
```

Comment:
The real API will return an empty 202 https://docs.cronofy.com/developers/api/events/bulk-delete-events/#example-response